ddd-kata.md

```diff
@@ -0,0 +1,94 @@
+* Perform a **Big Picture Event Storming** with your group in Mural or Miro
+* **Identify the bounded context and aggregates** within evaluated process in Mural or Miro
+* Establish the message flow from Bounded Context to Bounded Context using the **Domain Message Flow** Diagrams in Mural or Miro.
+* For each identified Bounded Context, specify using the **Bounded Context Canvas** in Mural or Miro.
+* Within the defined Bounded Contexts, defined the aggregates using the **Aggregate canvases** in Mural or Miro.
+
+Plan your time carefully. Timebox your efforts.
+
+### The Requirements
+
+Let’s assume you are a start-up, and you want to provide software for parking lots & garages for all major cities in Europe and for various park area providers.
+
+1. You can enter and leave the parking lot or the parking garage through several entrances and exits.
+2. There are specific parking spots for different types of vehicles (motorcycles, car, electric car, truck/bus, handicapped persons, family-friendly parking spots)
+3. Each parking spot has a unique identifier (for parking garage – number & floor or for parking lot – number &  area)
+4. The customer may collect a ticket at each entrance and leave via any exit.
+5. The ticket given to customer contains a parking spot ID where to park
+6. The ticketing information must be stored for 10 years due to tax reasons.
+7. There are three types of terminals:
+   1. The terminals at the entrances have a way for the customer to communicate the vehicle details to the system.
+   2. Customer drive into the parking garage / lot via entrance terminals where the collect the tickets
+   3. The terminals near the exits have a way to pay the ticket.
+   4. Detail out the actual payment process like select payment method, put in credit card, enter PIN are not relevant here.
+8. The terminals at the exit gates collect the paid ticket.
+9. The ticket has a read/write magnetic stripe containing information including a position to park.
+10. The magnetic stripe can be changed by the terminals.
+11. Parking garages have a parking guiding system, that guides customers to the correct parking spot. The guiding system recognizes the vehicle and guides it to the right parking spot.
+12. Parking lots do not have such a system.
+
+As domain expert, please stay from requirements perspective within these guardrails. You can detail them out as much as you want.
+
+### The Process
+
+#### Define the Team
+
+![Team Setup](./images/team-setup.png)
+
+As a **Moderator / Facilitator**, your task is to organize the workshops and conduct the workshops.
+
+As a **Domain Expert**, your tasks are:
+
+* Mimics the expert for parking lot software & management
+* Takes business decisions, has the last say on requirements
+* Researches and creates requirements
+
+As a **Team Member** your tasks are:
+
+* Interview the Domain Expert
+* Create the deliverables
+* Participate actively in all workshops
+
+#### How we normally run
```

Comment:
I would rephrase the section title: "Plan your timeline"

And maybe add a sentence that this is the timeline we apply in our workshops.

Reply:
I don't think the timeline proposal makes sense outside of our internal setups. Assuming people start actually doing this individually, why should they do it in this format? I think a more helpful (and maybe more interesting) guidance would be "try to spend this and that hours on this and that tool and try to write a prototype after x hours and reflect on it".

Reply:
I would keep this as a recommendation.